src/components/GoldenEye/GoldenEye.tsx

```diff
@@ -171,7 +170,6 @@ const GoldenEye = () => {
                 />
                 <Button type={"submit"}>Launch Dos Attack</Button>
                 {loading && <Alert children={"Lauching Dos attack against" + form.values.url}></Alert>}
```

Comment:
This should be handled by the HandleProcessData. No Adhoc methods of communication should be used elsewhere for consistency purposes.

- [ ] Remove alert and use controlled output formatting.

Reply:
Removed alert